cmd/bucket-targets.go

```diff
@@ -98,24 +98,20 @@ func (sys *BucketTargetSys) heartBeat(ctx context.Context) {
 		select {
 		case <-hcTimer.C:
 			sys.hMutex.RLock()
-			var eps []madmin.ServerProperties
+			eps := make([]madmin.ServerProperties, 0, len(sys.hc))
 			for _, ep := range sys.hc {
 				eps = append(eps, madmin.ServerProperties{Endpoint: ep.Endpoint, Scheme: ep.Scheme})
```

Comment:
```suggestion
			for i, ep := range sys.hc {
				eps[i] = madmin.ServerProperties{Endpoint: ep.Endpoint, Scheme: ep.Scheme}
```

Reply:
@poornas notice I am not allocating - I am only indicating the length of the slice

> +                       eps := make([]madmin.ServerProperties, 0, len(sys.hc))

It is to let Go know the final length so that it can optimize the slice allocation.

Reply:
missed that!